hack/jenkins/e2e.sh

```diff
@@ -180,12 +176,6 @@ GCE_PARALLEL_FLAKY_TESTS=(
     "Namespaces.*should\sdelete\sfast"
     "PD"
     "ServiceAccounts"
-    "Services.*change\sthe\stype"
-    "Services.*functioning\sexternal\sload\sbalancer"
-    "Services.*identically\snamed"
-    "Services.*release.*load\sbalancer"
-    "Services.*endpoint"
-    "Services.*up\sand\sdown"
```

Comment:
https://goto.google.com/k8s-test/job/kubernetes-e2e-gce-parallel-flaky/7619/ shows a failure of this test.

There were also PD test failures on the same build; not sure if related.